cmd/payledger/payledger.go

```diff
@@ -283,6 +297,98 @@ func buildLedger(node INode, config *Config) error {
 	return nil
 }
 
+func buildLedgerByTxData(config *Config) error {
+	csvFile, _ := os.Open(config.GenesisDataPath)
```

Comment:
why additional `.cvs` file? we can use a `.go` file to setup the initial data from the literals directly so that you don't have to worry about create a reader to load the `cvs` file && how to check the format etc.